README.md

```diff
@@ -3,40 +3,76 @@
   <img width=600 src="https://user-images.githubusercontent.com/5758427/231207493-62676aea-4cb9-4bb4-92b0-20309c8a933a.png">
 </p>
 <hr/>
+
 <p align="center">
-  <a href="https://docs.zama.ai/concrete"> 📒 Read documentation</a> | <a href="https://zama.ai/community"> 💛 Community support</a> | <a href="https://github.com/zama-ai/awesome-zama"> 📚 FHE resources</a>
+  <a href="https://docs.zama.ai/concrete"> 📒 Documentation</a> | <a href="https://zama.ai/community"> 💛 Community support</a> | <a href="https://github.com/zama-ai/awesome-zama"> 📚 FHE resources</a>
 </p>
 <p align="center">
 <!-- Version badge using shields.io -->
   <a href="https://github.com/zama-ai/concrete/releases">
     <img src="https://img.shields.io/github/v/release/zama-ai/concrete?style=flat-square">
   </a>
-  <!-- Link to tutorials badge using shields.io -->
-  <a href="#license">
+  <!-- License badge -->
+  <a href="#-license">
     <img src="https://img.shields.io/badge/License-BSD--3--Clause--Clear-orange?style=flat-square">
   </a>
 <!-- Zama Bounty Program -->
   <a href="https://github.com/zama-ai/bounty-program">
     <img src="https://img.shields.io/badge/Contribute-Zama%20Bounty%20Program-yellow?style=flat-square">
   </a>
 </p>
-<hr/>
 
-**Concrete** is an open-source FHE Compiler which simplifies the use of fully homomorphic encryption (FHE).
 
-FHE is a powerful cryptographic tool, which allows computation to be performed directly on encrypted data without needing to decrypt it first. With FHE, you can build services that preserve privacy for all users. FHE is also great against data breaches as everything is done on encrypted data. Even if the server is compromised, in the end no sensitive data is leaked.
 
-Since writing FHE programs can be difficult, Concrete, based on LLVM, make this process easier for developers.
+## Table of Contents
+- **[About](#about)**
```

Comment:
do you generate this ToC by hand, or do you use some tools? in the latter case, it might be interesting to keep the command line, such that we can refresh as often as needed.

Reply:
ok, will be complicated to maintain maybe